applicationset/services/internal/github_app/try.go

```diff
@@ -0,0 +1,42 @@
+package github_app
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/google/go-github/v35/github"
+)
+
+func Apps[T any](apps ...T) (result []One[T]) {
+	for _, app := range apps {
+		result = append(result, One[T]{App: app})
+	}
+	return result
+}
+
+type One[T any] struct {
+	App           T
+	lastAuthError time.Time
+}
+
+// Each tries each app in the list, and returns the first success.
+// If an app fails with a 401, it will be skipped for the next minute.
+func Each[T any, R any](apps []One[T], f func(service T) (R, error)) (result R, err error) {
```

Comment:
Is this an artifact of a previous iteration? Might just be GoLand, but as far as I can tell this is unused.